homeassistant/components/almond/strings.json

```
@@ -8,8 +8,8 @@
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Almond account.",
-      "cannot_connect": "Unable to connect to the Almond server.",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%].",
```

Sorry missed the period at the end, my bad.
```suggestion
      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
```